.github/workflows/ci.yml

```diff
@@ -59,42 +59,11 @@ jobs:
           pip3 install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
           pip3 install cmake ninja
           echo "/home/runner/.local/bin" >> $GITHUB_PATH
-      - name: Install AWS-SDK-CPP on Windows for S3 IO datapipes
```

Comment:
Could you please add `submodules: true` to `actions/checkout@v2` at line 55?

Reply:
Added
```
with:
  submodules: recursive
```
instead of `true`, since `aws-sdk-cpp` has recursive dependencies.